unfoldingWord/translationCore
# Add UDB to Scripture Pane

#### This pull request addresses:

Write a few sentences about what your pull request addresses, the relevant issue numbers, etc.
Issue #667 there is also a PR in the tC Apps repository under js_UDB


#### How to test this pull request:
Load a project like normal, see if the UDB shows up

Include testing instructions so that other developers know what to look for, and how to make sure that your feature/bugfix/enhancement really works.

https://github.com/translationCoreApps/ScripturePane/pull/19


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/797)
